package.json

```diff
@@ -14,11 +14,17 @@
   "repository": "https://github.com/hyperledger/transact-sdk-javascript.git",
   "license": "Apache-2.0",
   "scripts": {
+    "postinstall": "npm run generate-proto-files",
```

Comment:
Not sure what this line reordering is doing here. Maybe it got into the wrong commit?